Formula/ipv6toolkit.rb

```diff
@@ -6,6 +6,12 @@ class Ipv6toolkit < Formula
   license "GPL-3.0-or-later"
   head "https://github.com/fgont/ipv6toolkit.git"
 
+  livecheck do
+    url "http://pages.cs.wisc.edu/~plonka/ipv6toolkit/"
+    strategy :page_match
```

Comment:
```suggestion
```

Reply:
yeah, I was on the fence, glad that we dont need to specify it.